Sources/Testing/Testing.docc/OrganizingTests.md

```diff
@@ -82,47 +82,47 @@ Are equivalent to:
 }
 ```
 
-## Constraints on test suite types
+### Constraints on test suite types
 
-If a type is used as a test suite, it is subject to some constraints that are
+When using a type as a test suite, it's subject to some constraints that are
 not otherwise applied to Swift types.
 
-### An initializer may be required
+#### An initializer may be required
```

Comment:
I thought we didn't want H4s?

Reply:
We don't, but there's still an overarching H3 (which is technically an H2). The primary goal is to flatten the structure so there are no subsections, but we can work on that later. Under DocC, the Overview heading is `## Overview` and the sections underneath are `### Section title`.